src/components/ExecutorDetails.js

```diff
@@ -0,0 +1,311 @@
+import ExecutorDetailsSkeleton from '../skeletons/ExecutorDetailsSkeleton';
+import RunFailed from './Alerts/RunFailed';
+
+import { PermissionContext } from '../App';
+
+const ExecutorDetails = ({
+    match: { params: { executor_id, run_id, id }},
+    remediation,
+    playbookRun,
+    playbookRunSystems,
+    playbookRunSystemDetails,
+    getPlaybookRun,
+    getPlaybookRunSystems,
+    getPlaybookRunSystemDetails,
+    onCollapseInventory,
+    loadRemediation
+}) => {
+    const [ executor, setExecutor ] = useState({});
+    const [ systems, setSystems ] = useState([]);
+    const [ InventoryTable, setInventoryTable ] = useState();
+    const [ page, setPage ] = useState(1);
+    const [ pageSize, setPageSize ] = useState(50);
+    const [ refreshInterval, setRefreshInterval ] = useState();
+    const inventory = useRef(null);
+
+    const loadInventory = async () => {
+        const {
+            inventoryConnector,
+            mergeWithEntities,
+            INVENTORY_ACTION_TYPES
+        } = await insights.loadInventory({
+            react: React,
+            reactRouterDom,
+            reactCore,
+            reactIcons,
+            pfReactTable
+        });
+
+        getRegistry().register({
+            ...mergeWithEntities(reducers.playbookActivityIntentory({
+                INVENTORY_ACTION_TYPES, renderStatus: (status) => (<div className="ins-c-remediations-status-bar">
+                    { renderStatus(normalizeStatus(status)) }
+                </div>)
+            })())
+        });
+
+        const { InventoryTable } = inventoryConnector();
+        setInventoryTable(() => InventoryTable);
+    };
+
+    const onRefresh = (options) => {
+        if (inventory && inventory.current) {
+            setPage(options.page);
+            setPageSize(options.per_page);
+            inventory.current.onRefreshData(options);
+        }
+    };
+
+    useEffect(() => {
+        loadInventory();
+        loadRemediation(id);
+        getPlaybookRun(id, run_id);
+
+    }, []);
+    useEffect(() => {
+        if (playbookRun && playbookRun.data) {
+            setExecutor(playbookRun.data.executors.find(executor => executor.executor_id === executor_id) || {});
+        }
+
+        getPlaybookRunSystems(id, run_id, executor_id);
+    }, [ playbookRun ]);
+
+    useEffect(() => {
+        setSystems(() => playbookRunSystems.map(({ system_id, system_name, status }) => ({
+            id: system_id,
+            display_name: system_name,
+            status,
+            children: <Skeleton size='lg' />
+        })));
+    }, [ playbookRunSystems ]);
+
+    useEffect(() => {
+        setSystems(() => ([
+            ...systems.filter(system => system.id !== playbookRunSystemDetails.system_id),
+            { ...systems.find(system => system.id === playbookRunSystemDetails.system_id),
+                isOpen: true,
+                children: <SyntaxHighlighter language="yaml" style={ dark }>
```

Comment:
This is great! When we're in the running state and the console is being updated using the interval would it be possible to add some indication that it is live? (e.g. a small spinner below the console?) I know it's not in the mocks just curious if you had any ideas